itest/assets_test.go

```diff
@@ -882,44 +896,62 @@ func payInvoiceWithAssets(t *testing.T, payer, rfqPeer *HarnessNode,
 		sendReq.MaxShardSizeMsat = 80_000_000
 	}
 
+	var rfqBytes []byte
+	manualRfq.WhenSome(func(i rfqmsg.ID) {
+		rfq := manualRfq.UnsafeFromSome()
```

Comment:
Just use `i` here. That's the whole point of `WhenSome()`, that you don't have to use `UnsafeFrom()`.